src/pages/Games.js

```diff
@@ -1,17 +1,19 @@
 import "../styles/pages/Games.css"
-import {games} from "../data/content";
-import {GameCard} from "../components/games/GameCard";
+import { games } from "../data/content";
+import { GameCard } from "../components/games/GameCard";
 
 export const Games = () => {
     return (
         <div className="games-root">
             <h1 className="games-title">Games</h1>
             <div className="games-content">
                 {
-                    games.map(game => {
+                    games.map((game, index) => {
                         return (
-                            <GameCard game={game} />
-                        )
+                            <section id={`game-${index}`} key={game.urlTerm} style={{ padding: '50px 0' }}>
```

Comment:
Can u remove this padding, it doesn't [look good](https://acm-fun-git-fork-sp9324-master-acm-fun.vercel.app/games)